spec/mobility/backends/active_record/action_text_spec.rb

```diff
@@ -0,0 +1,510 @@
+require "spec_helper"
+
+return unless defined?(ActiveRecord)
+
+describe "Mobility::Backends::ActiveRecord::ActionText", orm: :active_record, type: :backend do
```

Comment:
This is a copy of `key_value_spec.rb`, where some specs which are not relevant to this scenario are commented out (e.g. when combining string and text types, as there is no need to configure such option in this case).